src/app.js

```diff
@@ -123,13 +123,13 @@ export default function(app) {
 
   function setElementData(element, name, value, oldValue) {
     if (name === "key") {
-    } else if ((name = name.toLowerCase()) === "style") {
+    } else if (name === "style") {
       for (var i in merge(oldValue, (value = value || {}))) {
         element.style[i] = value[i] || ""
       }
     } else {
       try {
-        element[name] = value
+        element[name.toLowerCase()] = value
```

Comment:
@madebyherzblut This only sets props, so attrs are not even touched now, which is what we wanted of course! 💥